www/include/configuration/configObject/host/listHost.php

```diff
@@ -241,7 +241,7 @@
 if (!$centreon->user->admin) {
     $aclFrom = ", {$aclDbName}.centreon_acl acl";
     $aclCond =
-        ' AND h.host_id = acl.host_id AND acl.service_id IS NULL '
```

Comment:
On my platform, service_id is NULL
I don't understand when it can be 0, cause default value of this column is NULL